plugins/tracker-resources/src/components/CreateIssue.svelte

```diff
@@ -73,6 +79,18 @@
       sort: { rank: SortingOrder.Ascending }
     }
   )
+
+  $: projectsQuery.query(
```

Comment:
Please move this logic to ProjectSelector popup, since we do not need to retrieve a list of projects if user not want to specify any.

Reply:
Done.